benchmarks/README.md

```diff
@@ -0,0 +1,62 @@
+# JMH benchmarks
+
+A collection of JMH benchmarks which may be useful for measuring Armeria performance.
+
+## Options
+
+- `-Pjmh.include=<pattern>`
+  - The benchmarks to run. All benchmarks if unspecified.
+    - `DownstreamSimpleBenchmark`
+    - `DownstreamSimpleBenchmark.emptyNonBlocking`
+- `-Pjmh.params=<spec>`
+  - The benchmark parameters. Uses the parameters specified in the benchmark code if unspecified.
+    - `clientType=NORMAL`
+    - `num=10,100;flowControl=false`
+- `-Pjmh.fork=<integer>`
+  - The number of forks. `1` if unspecified.
+- `-Pjmh.iterations=<integer>`
+  - The number of iterations. JMH default if unspecified.
+- `-Pjmh.warmupIterations`
+  - The number of iterations. Uses the value of `jmh.iterations` if unspecified.
+- `-Pjmh.profilers=<spec>`
+  - The profiler settings. Profiler disabled if unspecified.
+    - `jmh.extras.Async:asyncProfilerDir=...;flameGraphDir=...`
+- `-Pjmh.threads=<integer>` - The number of threads. JMH default if unspecified.
```

Comment:
nit: Newline between the title and its description, like the other items?
```
- `-Pjmh.threads=<integer>` 
  - The number of threads. JMH default if unspecified.
```

Reply:
Fixed